docs/how-to/configure-cephfs-mirroring.rst

```diff
@@ -0,0 +1,115 @@
+==================================
+Configure CephFS replication
+==================================
+
+MicroCeph replication framework supports enabling replication for CephFS workloads to a remote MicroCeph cluster.
```

Comment:
Suggest to add a sentence or two what this would be useful for and what gets replicated. Something like

> The MicroCeph replication framework supports enabling replication for CephFS workloads to a remote MicroCeph cluster. This can be for instance be used to replicate CephFS volumes or directories to a remote site for disaster recovery.

Reply:
+1